tool/tbot/botfs/fs_linux.go

```diff
@@ -58,12 +58,17 @@ var missingSyscallWarning sync.Once
 
 // openSecure opens the given path for writing (with O_CREAT, mode 0600)
 // with the RESOLVE_NO_SYMLINKS flag set.
-func openSecure(path string) (*os.File, error) {
+func openSecure(path string, write bool) (*os.File, error) {
+	mode := ReadMode
+	if write {
+		mode = WriteMode
+	}
```

Comment:
nit: Why not just pass the mode as an argument? You could define a type `type OpenMode int` to make it more explicit.
```suggestion
func openSecure(path string, mode OpenMode) (*os.File, error) {
```
I think that 
```
openStandard(path, WriteMode)
```
is easier to understand.

Reply:
Good thought, this is needlessly constrained.